charts/ingress-nginx/values.yaml

```diff
@@ -920,6 +920,9 @@ tcp: {}
 udp: {}
 #  53: "kube-system/kube-dns:53"
 
+# -- Prefix for TCP and UDP ports names
```

Comment:
I'm fine with this PR, and will leave it approved but please be more descriptive here: why this field is required? Why can this be used?

Maybe something like "Prefix for TCP and UDP port names in services. Some cloud providers, like yandex require a prefix like "port-something"

Reply:
Hi @rikatz ,

Thank you for your reply.
I fixed the comment to contain more info about why this preffix may be used

https://github.com/kubernetes/ingress-nginx/blob/804b87626833c8b445980b097b0a3dceb6670e08/charts/ingress-nginx/values.yaml#L923-L924